bangazonapi/models/product.py

```diff
@@ -15,7 +15,7 @@ class Product(SafeDeleteModel):
     customer = models.ForeignKey(
         Customer, on_delete=models.DO_NOTHING, related_name='products')
     price = models.FloatField(
-        validators=[MinValueValidator(0.00), MaxValueValidator(10000.00)],)
+        validators=[MinValueValidator(0.00), MaxValueValidator(17500.00)],)
```

Comment:
search the cohort slack channel for `clean_fields` and read the discussion there.

there's a bit more that's needed to enforce the validation.